web_development_101/the_basics/gearing_up.md

```diff
@@ -74,6 +74,14 @@ You will inevitably get stuck at some point in the curriculum, perhaps due to a
 
 You will have more success with Odin by putting **consistent** time into it rather than working on it once a week. Building a habit of studying every day at a specific time and with a specific goal will ensure that you make consistent progress.
 
+It may take you longer than others to grasp concepts, or it may take you less time. This doesn't mean you're smarter or dumber than others, it means you've had differing life experiences that may or may not have prepared you for learning this stuff. Someone who grew up around an engineer may have some advantages over someone who didn't, but it doesn't mean you can't learn those skills.
+
+The Odin Project isn't like college or university, TOP is self paced and allows you to get a really solid grasp of concepts before moving on. In school, you're forced to keep up or you will fail. The difference here is that coming into TOP, you're not expected to have much knowledge, there's no prerequisites. We've had people be successful coming through here that only knew how to check their email with a computer, we've also seen success from computer science degree holders. Treating TOP like a static timeline is ignorant simply because you don't know what you don't know yet, and that's OK! There's no due-dates on things in The Odin Project so you can spend the time to do it right and discuss the topics.
```

Comment:
> TOP
I would reword anywhere you have the TOP acronym, just because I haven't seen it used in the curriculum anywhere else. If I'm mistaken, disregard.
> We've had people be successful coming through here that only knew how to check their email with a computer, we've also seen success from computer science degree holders
I would either make these separate sentences or add an "and" after the comma. If you don't like that flow, changing the comma to a semi-colon fixes the grammatical issue.
> is ignorant simply
I'm not sure what it would be just yet, but I think just using an alternate word with more positive connotations would be better here.

Reply:
I have fixed the first two points, as for the last one:

> Treating The Odin Project like a static timeline is understandable, but is a sign of misplaced expectations, simply because you don't know what you don't know yet, and that's OK!

Does this sound OK?

Reply:
I like it, the sentence is just too long. I think breaking it up into different ones like

> Treating The Odin Project like a static timeline is understandable, but is a sign of misplaced expectations. You simply don't know what you don't know yet, and that's OK!

Reads a little better.

Reply:
Done.